pkg/catalog/manager/traverse.go

```diff
@@ -273,6 +293,20 @@ func (m *Manager) traverseAndUpdate(helm *helmlib.Helm, commit string, cmt *Cata
 		errstrings = append(errstrings, fmt.Sprintf("failed to update templates. Multiple error(s) occurred: %v", updateErrors))
 	}
 	if len(errstrings) > 0 {
+		if isChartInvalidErrors(InvalidChartErrors, createErrors, updateErrors) {
+			maxTemplateNames := 5
+			forgetErrorsLen := len(InvalidChartErrors)
+			if forgetErrorsLen < 5 {
+				maxTemplateNames = forgetErrorsLen
+			}
+			errorMessage := fmt.Sprintf("%s .... %v charts with invalid template url or name", strings.Join(errTemplateNames[:maxTemplateNames], " ; "), forgetErrorsLen)
```

Comment:
maybe say actual number of templates failed, but 5 shown if forget is more than max